### PR TITLE
Add landing message #1 #2

### PR DESCRIPTION
**Pull request by Syahirah2**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+    <h1>Welcome to Potionfy! Enjoy your stay!</h1>
```
Comment (Syahirah1) [inline index.html:10]: Could you remove the 'Enjoy your stay.' string and change 'Welcome to Potionfy' string to red?

**Comment by Syahirah2** [comment]
Sure, can do!

File: index.html (hunk 1 above)
Comment (Syahirah2) [reply index.html:10]: Sure, can do!

**Review by Syahirah2 — COMMENTED** [review]
Please change my code as per my comments on the review

**Pull request merged**